threatminer/plugin.spec.yaml

```
@@ -6,7 +6,7 @@ title: Threat Miner
 description: "Threat Miner is an open source search engine for fast threat intelligence research and pivoting with
   context. With the Threat Miner plugin for Rapid7 InsightConnect, users can lookup various pieces of information for
 threat intelligence gathering"
-version: 1.0.1
+version: 1.0.2
```

```suggestion
version: 2.0.0
```
regenerate after change

---

hmm, @zyoutz-r7 you were suggesting to use v1.0.2 as there is no logic change in the plugin, not sure how to resolve it

---

I saw the other comment explaining the reason, please ignore this one